recipes-tag/11-id-chx-analysis/meta.yaml

```diff
@@ -7,10 +7,11 @@ package:
   version: {{ year }}{{ cycle }}.{{ version }}
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   run:
     - chxtools
     - eiger-io
     - hdf5-lz4
+    - chxanalys
```

Comment:
conda-forge: dask, theano, graphviz, boto3
defaults: reportlab

Is [this](https://github.com/kennethreitz/maya) the maya @yugangzhang needs?